revive/config.toml

```diff
@@ -0,0 +1,29 @@
+ignoreGeneratedHeader = false
+severity = "warning"
+confidence = 0.8
+errorCode = 0
+warningCode = 0
```

Comment:
Will this fail CI if we violate the linter? I don't know if I really understand what errorCode and warningCode configure, so maybe these don't effect the action.

Reply:
Let me double check. I copied this from their proposed config but forgot to follow up on it.

Reply:
Good catch on these I changed them to return `1`.